script.js

```diff
@@ -20,4 +20,181 @@ function toggleNav() {
+    modalSeeButtonText: 'See Live',
+    modalSeeButtonIcon: '',
+    modalSourceButtonText: 'See Source',
+    modalSourceButtonIcon: '',
+  },
+  {
+    cardClass: 'works-card card4 card-reverse',
+    cardTitle: 'Multi-Post Stories',
+    cardDesktopImage: 'img/Snapshoot-Portfolio4.png',
+    cardMobileImage: 'img/Snapshoot-Portfolio-Mobile4.png',
+    company: 'CANOPY',
+    job: 'Back End Dev',
+    year: '2015',
+    cardInfo: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    htmlTag: 'html',
+    cssTag: 'css',
+    javascriptTag: 'javascript',
+    githubTag: 'github',
+    rubyTag: 'ruby',
+    bootstrapTag: 'bootstrap',
+    buttonText: 'See More',
+    modalInfo: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry.',
+    modalSeeButtonText: 'See Live',
+    modalSeeButtonIcon: '',
+    modalSourceButtonText: 'See Source',
+    modalSourceButtonIcon: '',
+  },
+];
+
+const portfolioSection = document.getElementById('portfolio');
+
+Array.from(portfolioSection.children).forEach((child, index) => {
+  child.innerHTML = `
+  <div class="${portfolio[index].cardClass}">
+    <div class="card-img card1">
+      <img class="work-img mobile" src="${portfolio[index].cardMobileImage}" alt="">
+      <img class="work-img desktop" src="${portfolio[index].cardDesktopImage}" alt="">
+    </div>
+    <div class="card-text">
+      <h2 class="card-title">${portfolio[index].cardTitle}</h2>
+      <ul class="card-details">
+        <li class="company">${portfolio[index].company}</li>
+        <li class="circle"><img src="img/circle.png" alt="circle"></li>
+        <li class="job">${portfolio[index].job}</li>
+        <li class="circle"><img src="img/circle.png" alt="circle"></li>
+        <li class="year">${portfolio[index].year}</li>
+      </ul>
+      <p class="card-info">${portfolio[index].cardInfo}</p>
+      <ul class="tags">
+        <li class="tag">${portfolio[index].htmlTag}</li>
+        <li class="tag">${portfolio[index].cssTag}</li>
+        <li class="tag">${portfolio[index].javascriptTag}</li>
+      </ul>
+      <button class="btn card-button" id="${[index]}">${portfolio[index].buttonText}</button>
+    </div>
+  </div>
+ `;
+});
```

Comment:
### In this part of the modal to match the Figma design:
- [ ] Kindly add the gray horizontal line that exists in the Figma design.
- [ ] Please for the button increase the space between the text and the icon.
- [ ] Please increase the bottom spacing between the modal and the page.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/188655199-3055960c-c85b-4bf1-b5a0-8709e59a14f9.png) | ![image](https://user-images.githubusercontent.com/24703171/188654941-a47c676b-4ad9-442e-93cb-5c08adb43bac.png) |